ragaai_catalyst/tracers/utils/rag_trace_json_converter.py

```
@@ -164,10 +164,30 @@ def get_context(input_trace):
         except Exception as e:
             logger.error(f"Error while extracting context from trace: {str(e)}")
             return ""
-        
+    
+    def get_span_errors(input_trace):
+        try:
+            if tracer_type == "langchain":
+                span_errors = {}
+                for span in input_trace:
+                    try:
+                        if "status" in span.keys() and span.get("status", {}).get("status_code", "").lower() == "error":
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)